integration-tests/integration/coordinator/remove-involved-staff.spec.js

```diff
@@ -136,16 +136,16 @@ context('A use of force coordinator can remove involved staff', () => {
       .then(reportId => cy.task('submitStatement', { userId: 'TEST_USER', reportId }))
       .then(() => cy.reload())
 
-    reportPage.deleteInvolvedStaff('TEST_USER').should('not.be.visible')
+    reportPage.deleteInvolvedStaff('TEST_USER').should('not.exist')
     reportPage.deleteInvolvedStaff('MRS_JONES').should('be.visible').click()
```

Comment:
Are these required due to new version of Cypress 6.3.0?

Reply:
yes they are! Very confusing